build.sbt

```diff
@@ -47,6 +45,22 @@ lazy val zapAutomation = Project(appName, file("."))
     makePublicallyAvailableOnBintray := true,
     scalaVersion := "2.11.12",
     libraryDependencies ++= compileDependencies ++ testDependencies,
-    crossScalaVersions := Seq("2.11.12"),
+    libraryDependencies := {
+      CrossVersion.partialVersion(scalaVersion.value) match {
```

Comment:
Just wondering if there is a reason we need to break out the dependencies for the different scala versions? Looks like those two play dependencies are cross-compiled for `2.12` so would simplify the build if we don't need it

Reply:
@bennetimo agreed, I've reverted the dependency changes.